recipes/mailer/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set name = "mailer" %}
+{% set version = "0.8.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  sha256: 3411a12197731e0d5379ab194d9acc8d0d71c8b95fdfb11474d67c3e9860070e
+
+build:
+  number: 0
```

Comment:
Just one thing here, is it possible to this recipe be ``noarch: python`` ?

Reply:
It looks like a python project for me

Reply:
No, unfortunately it uses 2to3 during installation time

Reply:
Ah, ok then! I'm going to merge this!

Thanks to double check that  :)